firedrake/slate/slac/utils.py

```diff
@@ -211,6 +445,32 @@ def topological_sort(exprs):
     return schedule
 
 
+# Adapted from tsfc.gem.node.py
```

Comment:
So what is the difference? That `children` are called `operands` in SLATE? Is there anything else?

Reply:
Yes, that is the only difference.

Reply:
For example, both `post_traversal` and `traverse_dags` are copy-paste from `gem.node`, replacing `children` with `operands`.  If SLATE nodes merely had `.children`, you could just import these functions.  `gem.node` does not even require `.children` to be a slot or an attribute, the tuple of children can also be constructed as an property, e.g.
```python
    @property
    def children(self):
        return self.operands
```
on the base SLATE node.